salt/kubelet/kubelet-config.jinja

```diff
@@ -0,0 +1,61 @@
+kind: KubeletConfiguration
+apiVersion: kubelet.config.k8s.io/v1beta1
+{%- if grains['lsb_distrib_id'] == "CAASP" %}
+staticPodPath: "/etc/kubernetes/manifests"
+{%- endif %}
+syncFrequency: 1m0s
+fileCheckFrequency: 20s
+httpCheckFrequency: 20s
+address: 0.0.0.0
+port: {{ pillar['kubelet']['port'] }}
+tlsCertFile: "/var/lib/kubelet/pki/kubelet.crt"
+tlsPrivateKeyFile: "/var/lib/kubelet/pki/kubelet.key"
+authentication:
+  x509:
+    clientCAFile: "{{ pillar['ssl']['ca_file'] }}"
+  webhook:
+    enabled: false
+    cacheTTL: 2m0s
+  anonymous:
+    enabled: true
```

Comment:
`authentication:anonymous:enabled == true`?

Reply:
Ups, fixed

Reply:
I think this we would like to keep it. I have checked and we didn't any authentication and authorization method to Kubelet, but I'm testing this. For more details please look on https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-authentication-authorization/

Reply:
As I have checked on our current configuration, this value is default for us. We don't use any authentication and authorization method to Kubelet.